01-HTML-CSS-Basics/2. CSS/2.5 Layout/index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN">
+<HTML>
```

Comment:
Usually tags are written in lowercase, but more a recommendation than an error here :)